features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/ResourcesLRUCache.kt

```diff
@@ -125,26 +121,18 @@ internal class Base64LRUCache(
     }
 }
 
-internal data class CacheData(val base64Encoding: ByteArray, var resourceId: ByteArray?) {
+internal data class CacheData(val resourceId: ByteArray) {
```

Comment:
do we need a class with only single property? seems like we can get rid of it.

Reply:
Good point. Removing the base64 indeed removes our need for this container class